core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -311,21 +311,22 @@ public void getFileInfo() throws Exception {
     long fileId;
     FileInfo info;
 
-    fileId = mFileSystemMaster.getFileId(ROOT_URI);
+    fileId = mFileSystemMaster.getFileId(ROOT_URI);//get the ROOT_URI's id
```

Comment:
I think the purpose of this task was to add a method-level Javadoc ... I don't find these new comments helpful or necessary